protostar/starknet_gateway/gateway_facade_input_validation_test.py

```diff
@@ -0,0 +1,284 @@
+import pytest
+from .gateway_facade_input_validation import (
+    JSONParsingErrorException,
+    validate_deploy_input,
+    InvalidInputException,
+    get_abi_from_json,
+)
+
+COMPILED_CONTRACT_ABI_ONLY = """
```

Comment:
```suggestion
COMPILED_CONTRACT_ABI = """
```
How should `ONLY` be understood?